src/components/PolarisTestProvider/PolarisTestProvider.tsx

```diff
@@ -59,19 +59,20 @@ const defaultMediaQuery: MediaQueryContextType = {
   isNavigationCollapsed: false,
 };
 
-export function PolarisTestProvider({
-  strict,
-  children,
-  i18n,
-  appBridge,
-  link,
-  theme = {},
-  mediaQuery,
-  features = {},
-  frame,
-}: PolarisTestProviderProps) {
-  const Wrapper = strict ? React.StrictMode : React.Fragment;
+export function PolarisTestProvider(props: PolarisTestProviderProps) {
+  const {
```

Comment:
I think keeping the original structure and having `features = {}` as a default prop is fine here